### ISSUE TITLE
Unable to add new control exception; file_id doesn't have a default value

### ISSUE THREAD

**Issue by vh-arctia** [simplerisk/code#46]
Hello,

I keep running into the following when attempting to create a new control exception:

```
[11-Aug-2024 23:32:10 Europe/Helsinki] PDOException: SQLSTATE[HY000]: General error: 1364 Field 'file_id' doesn't have a default value in /var/www/simplerisk/includes/governance.php:2874
Stack trace:
#0 /var/www/simplerisk/includes/governance.php(2874): PDOStatement->execute()
#1 /var/www/simplerisk/includes/api.php(7842): create_exception('test', 1, 0, 14, 1333, 11, '', '2024-08-11', 0, '2024-08-11', '0000-00-00', 0, 0, '', '', '')
#2 /var/www/simplerisk/vendor/raid-software/epiphany/src/EpiRoute.php(149): create_exception_api()
#3 /var/www/simplerisk/api/index.php(560): EpiRoute->run()
#4 {main}

```

I'm current running the following:

> You are running Application version 20240603-001
> You are running Database version 20240603-001

This is a fresh install.

**Comment by jsokol** [comment]
Can you please let me know which method you used to perform this installation?  I can't say that I've ever seen this message before.

**Comment by vh-arctia** [comment]
docker with DB_SETUP=automatic-only. Database is MariaDB 10.11.6 running on Debian 12.6. Full version string:

> mysql  Ver 15.1 Distrib 10.11.6-MariaDB, for debian-linux-gnu (x86_64) using  EditLine wrapper

**Comment by jsokol** [comment]
That could very well be the issue.  We've recently found that there are a number of subtle differences between MySQL and MariaDB that result in inconsistencies in the application.  Because of this, we have decided to no longer support MariaDB as a database with the pending new release.  I'll make sure to note that we need to update the documentation to remove any references to MariaDB, but in your case, you'll probably want to switch over to MySQL.  If you still have the issue after doing that, let us know.
